bin/install-jetpack.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+set -Eeuo pipefail
+
+# Check if nvm is installed
+if ! command -v nvm ; then
+  source ~/.nvm/nvm.sh &>/dev/null
+  if [ $? -ne 0 ]; then
+    echo "nvm is not installed or cannot be sourced from ~/.nvm/nvm.sh. Please install nvm and run this script again."
+    exit 1
+  fi
+fi
+
+# Set up node requirement for Jetpack
+pushd jetpack
+
+node_version=$(cat .nvmrc)
+nvm install "$node_version"
```

Comment:
Looks like we might not need the node version here. see https://github.com/nvm-sh/nvm/issues/318

Reply:
Also don't need `nvm use`, `nvm install` will switch to the specified version.